File: backend/app/schemas/media.py
from pydantic import BaseModel, Field

class UploadedImageRead(BaseModel):
    image_url: str
    public_id: str
    width: int | None = None
    height: int | None = None
    bytes: int | None = None

class DeleteUploadedImageRequest(BaseModel):
    public_id: str = Field(min_length=1, max_length=255)

File: mobile/types/media.ts
export type UploadedImage = {
    image_url: string;
    public_id: string;
    width: number | null;
    height: number | null;
    bytes: number | null;
};

export type LocalImageUpload = {
    uri: string;
    fileName?: string | null;
    mimeType?: string | null;
};